dev/tests/integration/framework/Magento/TestFramework/Helper/Amqp.php

```diff
@@ -13,26 +13,63 @@
  */
 class Amqp
 {
+    const CONFIG_PATH_HOST = 'queue/amqp/host';
+    const CONFIG_PATH_USER = 'queue/amqp/user';
+    const CONFIG_PATH_PASSWORD = 'queue/amqp/password';
+    const DEFAULT_MANAGEMENT_PORT = '15672';
+
     /**
      * @var Curl
      */
     private $curl;
 
+    /**
+     * @var \Magento\Framework\App\DeploymentConfig
+     */
+    private $deploymentConfig;
+
     /**
      * RabbitMQ API host
      *
      * @var string
      */
-    private $host = 'http://localhost:15672/api/';
+    private $host;
 
     /**
      * Initialize dependencies.
+     * @param \Magento\Framework\App\DeploymentConfig $deploymentConfig
      */
-    public function __construct()
-    {
+    public function __construct(
+        \Magento\Framework\App\DeploymentConfig $deploymentConfig
```

Comment:
For backward compatibility
```suggestion
        \Magento\Framework\App\DeploymentConfig $deploymentConfig = null
```

Reply:
@sivaschenko Thanks, good catch! Updated here https://github.com/magento/magento2/pull/18978/commits/dc8b17039f8be1e443182d671c1f2b4e113c49a5